robottelo/vm.py

```diff
@@ -162,10 +161,9 @@ def create(self):
                 u'Failed to run snap-guest: {0}'.format(result.stderr))
 
         # Give some time to machine boot
-        time.sleep(60)
-
         result = ssh.command(
-            u'ping -c 1 {0}.local'.format(self._target_image),
+            u'for i in {{1..60}}; do ping -c1 {0}.local && break; sleep 1;'
+            u' done; echo $?'.format(self._target_image),
```

Comment:
@kbidarkar I was thinking about trying to ssh in a loop instead of pinging the host. That would ensure we cut the loop after we are really able to connect to tcp/22.

your other comment is an interesting read, however i would avoid this solution if we can, as it involves changing the settings on the virtual host, which would require changes to the base image. We would loose flexibility with this as we might decide to dynamically deploy systems with various system, not necessarily using pre-built base images, etc.

what do you think?